ChangeLog

```diff
@@ -25,6 +25,7 @@
  * #977 (HypothesisTest::ChiSquared is bogus)
  * #982 (LinearModelTest::LinearModelDurbinWatson with several factors)
  * #983 (Bogus Normal parameter distribution)
+ * #1000 (SaltelliSensitivityAlgorithm with LowDiscrepancyExperiment produces wrong results)
```

Comment:
Maybe a comment concerning new Github Issues usage

Reply:
otherwise it is ok for me

Reply:
it's in the changelog that we use github issues now